proto/job.proto

```diff
@@ -80,11 +80,19 @@ message GetRayJobRequest {
 message ListRayJobsRequest {
   // Required. The namespace of the job to be retrieved.
   string namespace = 1 [(google.api.field_behavior) = REQUIRED];
-  // TODO: support paganation later
+
+  // A continue token to request the next page of results. The token is acquired
+  // from the previous ListRayJobs call or can be omitted when fetching the first page.
+  string continue = 2;
+  // The maximum number of jobs to return for the requested page.
+  // For backward compatibility, the default value is 0 which returns all jobs without pagination.
+  int64 limit = 3;
 }
 
 message ListRayJobsResponse {
   repeated RayJob jobs = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // The continue token for the next page of jobs.
```

Comment:
From client's perspective, how do they know they could stop issuing new requests?
(I am requesting for documentation on stop behavior)